xray/plot/plot.py

```diff
@@ -384,7 +435,11 @@ def _plot2d(plotfunc):
     Parameters
     ----------
     darray : DataArray
-        must be 2 dimensional.
+        Must be 2 dimensional
+    x : string, optional
+        Coordinate for x axis. If None use darray.dims[1]
```

Comment:
Which is actually kind of nice since the plot corresponds to the printed output of the array.